datastore/integration_test.go

```diff
@@ -484,6 +484,57 @@ func testSmallQueries(ctx context.Context, t *testing.T, client *Client, parent
 		x()
 	}
 }
+func TestIntegration_FilterEntity(t *testing.T) {
+	ctx := context.Background()
+	client := newTestClient(ctx, t)
+	defer client.Close()
+
+	parent := NameKey("SQParent", "TestIntegration_Filters"+suffix, nil)
+	now := timeNow.Truncate(time.Millisecond).Unix()
+	tomorrow := timeNow.Truncate(time.Millisecond).AddDate(0, 0, 1).Unix()
+	children := []*SQChild{
```

Comment:
Not needed. Here, SQChild is the kind of entity being used for testing. Any such structs can be used to work with the client

Reply:
Not a blocking concern, but another thing to clean up would be to not export the type in that case. Since this is a test nothing will every use that type outside of this package. But that can be something to address in future passes.

Reply:
Will do in subsequent PRs